frontend/src/popup.jsx

```diff
@@ -0,0 +1,46 @@
+import {toStringHDMS} from 'ol/coordinate';
+import {toLonLat} from 'ol/proj';
+import React, { useState, useEffect, useRef } from 'react';
+
+
+export default function PopupComponent(props) {
+  const {
+    location,
+    message,
+    overlay,
+    toggleEditMenu
+  } = props;
+
+  const closerElement = useRef();
+  const contentElement = useRef();
```

Comment:
I think I had these refs because I was copying [this openlayers example](https://openlayers.org/en/latest/examples/popup.html). Then translated the `document.getElementById` to `useRef` because I saw that in a different example with React. You're right it's totally unnecessary here!